tuf/data/errors.go

```diff
@@ -0,0 +1,22 @@
+package data
+
+import "fmt"
+
+// ErrInvalidMetadata is the error to be returned when metadata is invalid
+type ErrInvalidMetadata struct {
+	role string
+	msg  string
+}
+
+func (e ErrInvalidMetadata) Error() string {
+	return fmt.Sprintf("%s type metadata invalid: %s", e.role, e.msg)
+}
+
+// ErrMissingMeta - couldn't find the FileMeta object for a role or target
+type ErrMissingMeta struct {
+	Role string
```

Comment:
Nah, I'm just being pedantic. We need to overhaul our errors and make sure they're defined and returned in the right places. Packages importing and creating eachother's errors is probably a sign that we need to look at our code organization a bit more.